Wikipedia/Code/CLLocationManager+WMFLocationManagers.m

```diff
@@ -2,20 +2,29 @@
 
 @implementation CLLocationManager (WMFLocationManagers)
 
+
++ (instancetype)wmf_fineLocationManager {
+    CLLocationManager *locationManager = [[CLLocationManager alloc] init];
+    locationManager.desiredAccuracy = kCLLocationAccuracyBest;
+    locationManager.activityType = CLActivityTypeFitness;
+    locationManager.distanceFilter = 1;
+    return locationManager;
+}
+
 + (instancetype)wmf_coarseLocationManager {
     CLLocationManager *locationManager = [[CLLocationManager alloc] init];
     locationManager.desiredAccuracy = kCLLocationAccuracyKilometer;
     locationManager.activityType = CLActivityTypeFitness;
-    locationManager.distanceFilter = 10;
+    locationManager.distanceFilter = 1000;
```

Comment:
fixing… 10 was incorrect before… the fine one is for small distances. This is now actually corse… 1k

Its also needed now for the new logic